### ISSUE TITLE
[Cookies]network.cookie.cookieBehavior set to 1 is causing an issue

### ISSUE THREAD

**Issue by bitpixl** [ghacksuserjs/ghacks-user.js#452]
Hi,

I'm unable to open https://www.reddit.com/r/oddlysatisfying/comments/8tf7eg/opening_a_cattail_plant.
The gfycat link above the frame does work and the video does play there.

User.js version: (19th June 2018, latest)
Firefox version: 60.0.2
OS version: Windows 10 Pro x64
Extensions used: None
Clean/New profile: Yes

I've narrowed it down to "network.cookie.cookieBehavior", which is changed to "1" in my /*** 5000: PERSONAL [SETUP] section. When I change it back to 2, the video works again.

Can anyone explain why it doesn't work when it's set to 1 (allow same host), but does work when it's set to 2 (disallow all). It makes no sense for me as I see setting it to 1 should be less restrictive?

**Comment by Thorin-Oakenpants** [comment]
IDK. Weirdly enough, blocking all cookies but allowing a site exception works, but allowing first party doesn't.

**Comment by Thorin-Oakenpants** [comment]
Actually, the video can always play regardless of cookies (and cookies also control localStorage + IDB) - you just need to right click in the video and select `Play`. Its just the video controls that seem affected. Blame reddit.

**Comment by bitpixl** [comment]
Good catch! Can't believe I tried that myself. I still find it quite strange that it all works with "network.cookie.cookieBehavior" set to 2.

**Comment by Thorin-Oakenpants** [comment]
https://old.reddit.com/r/firefox/comments/96fyli/websites_not_working_properly_with_3rd_party/

**Comment by crssi** [comment]
When 3rd party blocked, there is some overlay element on gyfcat preventing to use play controls.
But this is just fast look into and will take more time than just a few seconds for investigation.

**Comment by Thorin-Oakenpants** [comment]
Yup. I understand that. See above where I said you could right-click and select `play`. The reddit post is the exact issue @bitpixl had. We just never bothered to narrow it down to a 3rd party cookie. Fuck cookies.

I'm just finding it more and more disgusting things break because of something. Either a cookie, or a referrer, or some other tracking mechanism. Soon you won't be able to do anything without a service worker. Time to unplug the net

**Comment by crssi** [comment]
Mostly some bullshit, not need to waste a time, videos doesn't work and its rare.
I am browsing now for a few weeks with not allowing 3rd party cookies by FF and its working just fine, except this reddit gfycat bullshit, which IMO is just wasting time like FB is.
By my uBO rules, those even doesn't show up. ;)

**Comment by Thorin-Oakenpants** [comment]
Same. I block most media anyway. But I'm not atypical of users.
